vendor/brave-ios/Ads/Generated/NativeAdsClientBridge.h

```diff
@@ -14,7 +14,7 @@
 - (void)confirmAction:(const std::string &)uuid creativeSetId:(const std::string &)creative_set_id confirmationType:(const ads::ConfirmationType &)confirmation_type;
 - (void)eventLog:(const std::string &)json;
 - (void)getCreativeAdNotifications:(const std::vector<std::string> &)categories callback:(ads::OnGetCreativeAdNotificationsCallback)callback;
-- (void)getAdConversions:(const std::string &)url callback:(ads::OnGetAdConversionsCallback)callback;
+- (void)getAdConversions:callback:(ads::OnGetAdConversionsCallback)callback;
```

Comment:
Have to remove `callback:` here (will fail to compile)

Reply:
Fixed